src/benchmarks/beluga_vs_nav2_multi_dataset/reports/nominal_report/sections/platform.inc

```diff
@@ -0,0 +1,131 @@
+Methodology
+-----------
+
+Datasets used
+^^^^^^^^^^^^^
+
+A number of pre-recorded datasets were selected to evaluate the performance of the Beluga AMCL and Nav2 AMCL localization systems.
+A number of public datasets were considered for this evaluation, looking for a diversity of environments and robots that were representative of a wide range of real-world scenarios.
+
+The following datasets were selected based on their characteristics and availability for this evaluation:
+
+.. list-table:: Real robot datasets
+   :widths: 25 70
+   :header-rows: 1
+
+   * - Name
+     - Description
+   * - `Willow Garage dataset <https://web.archive.org/web/20151207202459id_/http://cgi.cs.duke.edu/~mac/papers/iros12_sm.pdf>`_
+     - Large office environment with multiple recordings along different trajectories and times, including multiple rooms and hallways.
+   * - `TorWIC SLAM dataset <https://github.com/Viky397/TorWICDataset>`_
+     - Warehouse environment with multiple recordings along different trajectories and times.
+   * - `TorWIC Mapping dataset <https://github.com/Viky397/TorWICDataset>`_
+     - Trajectories in a synthetic warehouse-like environment undergoing incremental changes.
+   * - `Cartographer Magazino <https://github.com/magazino/cartographer_magazino>`_
+     - Two sample datasets for mapping and localization that were recorded in a hallway.
+   * - `OpenLORIS-Scene dataset (Office set) <https://lifelong-robotic-vision.github.io/dataset/scene.html>`_
+     - Short trajectories in an office environment.
+
+None of these datasets were originally conceived for the evaluation of 2D localization systems,
+and therefore none of them provided a complete ground-truth that included both the real-world trajectory and the environment's
+occupancy grid map.
+
+For this reason ground-truth data was generated locally from the bagfiles themselves using Cartographer SLAM to create mutually
+consistent ground-truth trajectory and occupancy grid map for each bagfile.
+
+It has to be notices that this approach has some drawbacks that need to be taken into account when interpreting the results later:
+
+- Both the space and time diversity of the original datasets is lost, as the ground-truth is generated for each individual bagfile representing the state of the environment as perceived when the data was recorded.
+- All of the objects present in the environment are part of the generated occupancy grid map; i.e. there are no unmapped objects.
+
+These real robot datasets have two more limitations worth mentioning:
```

Comment:
@glpuga nit: 
```suggestion
These real-world datasets have two more limitations worth mentioning:
```
or use `real robot` instead of `real world` everywhere else, for consistency.